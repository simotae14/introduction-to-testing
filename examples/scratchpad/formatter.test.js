import { test, expect } from 'vitest';
import { formatUser } from './formatter';

test('formats user information correctly', () => {
  const user = { name: 'Alice', age: 30 };
  const formattedUser = formatUser(user);
  expect(formattedUser).toMatchInlineSnapshot(`"User: Alice, Age: 30"`);
});
