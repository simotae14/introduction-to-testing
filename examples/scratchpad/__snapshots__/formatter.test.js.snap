// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`formats user information correctly 1`] = `"User: Alice, Age: 30"`;
